content/docs/user-guide/components/reference/atom/bloom.md

```diff
@@ -1,20 +1,39 @@
 ---
+linkTitle: Bloom
 title: Bloom Component
-linktitle: Bloom
-description: "Open 3D Engine (O3DE) Bloom component reference."
+description: The Bloom component, which simulates real-world light bleeding (glow), is provided by the Atom Gem in Open 3D Engine (O3DE). 
 toc: true
 ---
 
-The **Bloom component** simulates real-world light bleeding, or glow.
+The **Bloom** component creates *bloom*, a post-processing effect that simulates real-world light bleeding, or glow.
 
 
-## Provider ##
+## Provider
 
-[Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom/)
+[Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom)
+
+
+## Dependencies
+
+[PostFX Layer component](./postfx-layer)
 
 
 ## Properties
 
-{{< todo issue=https://github.com/o3de/o3de.org/issues/717 >}}
-Add properties.
-{{< /todo >}}
\ No newline at end of file
+![Bloom component interface](/images/user-guide/components/reference/atom/bloom-component-ui.png)
+
+
+### Bloom properties
+
+| Property | Description | Value | Default |
+| - | - | - | - |
+| **Enable Bloom** | If enabled, activates the bloom effect. | Boolean | `Enabled` |
+| **Overrides** | If enabled, all Bloom component properties will be set to the values specified in the **Overrides** property group. | Boolean | `Enabled` |
+| **Threshold** | Bloom effect applies only to pixels with a brightness value greater than this threshold. | Float: 0.0 to Infinity  | `1.0` |
+| **Knee** | Creates a gradual transition between the pixels below and above the threshold. This softens and spreads the bloom effect.  | Float: 0.0 - 1.0 | `0.5` |
+| **Intensity** | Scales the intensity of the bloom effect. | Float: 0.0 to Infinity | `0.5` |
```

Comment:
Happened to be in the Editor, so I loaded up the component and noticed that Intensity has a slider bar.
```suggestion
| **Intensity** | Scales the intensity of the bloom effect. | Float: 0.0 to 10000.0 | `0.5` |
```

Reply:
Some UI sliders have soft min and max values.  A user can type a value that is smaller or larger than dragging the slider would allow.  The slider's range is supposed to be set to sensible values, which may not be the case here.